src/controllers/FulfillmentController.js

```diff
@@ -37,6 +39,15 @@ class FulfillmentController {
       return fail(res, err);
     }
   };
+
+  closeTheLoop = async (event, context, callback) => {
+    try {
+      await this.followUpService.closeTheLoop(event, context, callback);
```

Comment:
You're calling closeTheLoop() on the followUpService, but it's defined in the fulfillmentService. I can't see how this would be working. Please provide me some data, and a script (instructions), so I can test this out myself.

Reply:
Oops, that is my mistake. I will repair now

Reply:
done